storagemanager/storage/models.py

```diff
@@ -8,3 +8,7 @@ class Item(models.Model):
 
     def __str__(self):
         return f'({self.category}) {self.type}, {self.model}, [SN: {self.serial_number}]'
+
+    def say_hello(self):
```

Comment:
It doesn't say hello, it rather introduces itself ;)

Also, it's not good practice to create methods only for the sake of tests. Tests should test existing features, and not opposite (features created only to satisfy tests).